sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -86,29 +86,9 @@ static RT::PiProgram createBinaryProgram(const ContextImplPtr Context,
 
   RT::PiProgram Program;
 
-  bool IsCUDA = false;
-
   // TODO: Implement `piProgramCreateWithBinary` to not require extra logic for
   //       the CUDA backend.
```

Comment:
I believe you didn't remove this `TODO` because it's still an issue.

Reply:
Yes, the `piProgramCreateWithBinary` is still not implemented in the CUDA backend (someone else is working on that patch, should be there soon!)